doc/api-extensions.md

```diff
@@ -2424,3 +2424,10 @@ The OVN driver will allocate IP addresses from the subnets specified in the upli
 
 Adds the ability to explicitly specify a trust token when creating a certificate
 and joining an existing cluster.
+
+## `instance_create_start`
+
+Adds a new `start` field to the `POST /1.0/instances` API which when set
```

Comment:
Sorry, nitpicking here. ;)

```suggestion
Adds a `start` field to the `POST /1.0/instances` API which when set
```

You wouldn't add an old field, would you? :wink:

Reply:
I agree. Fixed now :)